### PR TITLE
Remove DRMWorkaround for AES-256 in older Java

### PR DESCRIPTION
**Pull request by msgilligan**
Android and OpenJDK don't need this and I'm pretty sure that  Oracle JDK 9 and even recent builds of Oracle JDK don't need it either.

Maybe we should verify this before merging this PR or releasing this version.

### CODE REVIEW

**Comment by schildbach** [comment]
Are we even using the JCE API any more? We've got BouncyCastle.

File: core/src/main/java/org/bitcoinj/crypto/BIP38PrivateKey.java [hunk 1]
```diff
@@ -152,7 +152,6 @@ private ECKey decryptNoEC(String normalizedPassphrase) {
             byte[] key = Arrays.copyOfRange(derived, 32, 64);
             SecretKeySpec keyspec = new SecretKeySpec(key, "AES");
 
-            DRMWorkaround.maybeDisableExportControls();
             Cipher cipher = Cipher.getInstance("AES/ECB/NoPadding");
```
Comment (msgilligan) [inline core/src/main/java/org/bitcoinj/crypto/BIP38PrivateKey.java:155]: @schildbach I'm guessing this line uses JCE (or at least used to)

**Comment by schildbach** [comment]
It is my understanding that if you'd need the workaround but have not applied it,  your code (e.g. unit tests) would fail rather than silently yield wrong results. Under that assumption, I'd tend to merge this.

**Comment by msgilligan** [comment]
> Are we even using the JCE API any more? We've got BouncyCastle.

I think we are. See my comment in `BIP38PrivateKey.java`

**Comment by msgilligan** [comment]
> It is my understanding that if you'd need the workaround but have not applied it, your code (e.g. unit tests) would fail rather than silently yield wrong results. Under that assumption, I'd tend to merge this.

Yeah, I think we should just merge it.

File: core/src/main/java/org/bitcoinj/crypto/BIP38PrivateKey.java (hunk 1 above)
Comment (schildbach) [reply core/src/main/java/org/bitcoinj/crypto/BIP38PrivateKey.java:155]: Answering my own question: Grepping for `getInstance("` yields 8 occurences.

**Comment by schildbach** [comment]
According to the exception message in `DRMWorkaround`, the parts in bitcoinj that needed this are: Tor (which we don't have any more) and BIP38 (implemented via `BIP38PrivateKey.java`)

**Comment by schildbach** [comment]
Merged.

**Pull request closed without merging**